build.sbt

```diff
@@ -1,20 +1,18 @@
-import play.core.PlayVersion
+import AppDependencies._
+import ScoverageSettings.scoverageSettings
 import sbt.Keys.compile
 import uk.gov.hmrc.DefaultBuildSettings.{addTestReportOption, defaultSettings, scalaSettings}
-import wartremover.{Wart, Warts}
 import wartremover.WartRemover.autoImport.{wartremoverErrors, wartremoverExcluded}
+import wartremover.{Wart, Warts}
+
+val appName = "help-to-save-stub"
 
 lazy val appDependencies: Seq[ModuleID] = dependencies ++ testDependencies()
-lazy val scoverageSettings = {
-  import scoverage.ScoverageKeys
-  Seq(
-    // Semicolon-separated list of regexs matching classes to exclude
-    ScoverageKeys.coverageExcludedPackages := "<empty>;.*Reverse.*;.*config.*;.*(AuthService|BuildInfo|Routes).*",
-    ScoverageKeys.coverageFailOnMinimum := true,
-    ScoverageKeys.coverageHighlighting := true,
-    Test / parallelExecution := false
-  )
-}
+lazy val plugins: Seq[Plugins]          = Seq.empty
+lazy val playSettings: Seq[Setting[_]]  = Seq.empty
```

Comment:
I don't think these things are necessary though. Given they're empty list and we have a dedicated plugins file, I'd just leave them inlined.